lib/js/styles/ColorsList.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+	<div>
+		<div v-for="colorList in colorsLists" :key="colorList.title" class="colorDefinition">
+			<div class="colorsList">
+				<div class="colorsList__row">
+					<h2>{{ colorList.title }}</h2>
+					<ol class="colorsList__categories">
+						<li
+							v-for="(colors, colorCategory) in colorList.list"
+							:key="colorCategory"
+							class="colorsDefinition__category"
+						>
+							{{ colorCategory }}
+						</li>
+					</ol>
+				</div>
+				<div
+					v-for="(colors, colorCategory) in colorList.list"
+					:key="colorCategory"
+					class="colorsList__row"
+				>
+					<h3 :id="'category-' + colorCategory">{{ colorCategory }}</h3>
+					<div v-for="color in colors" :key="color.id" class="colorDefinition">
+						<div class="colorDefinition__id">{{ color.label }}</div>
+						<div class="colorDefinition__value">{{ color.value }}</div>
+						<div class="colorDefinition__color">
+							<span
+								v-if="isHexOrRGBA(color.value)"
+								:style="{ background: color.value }"
+							></span>
+							<span v-else :style="{ background: 'var(' + color.value + ')' }"></span>
```

Comment:
when we fix exported variabled, then this conditiona handling won't be necessary

Reply:
I would not want change json's structures:
```    {
      "id": "_raw-wnl.scss_theme-50",
      "label": "theme-50",
      "value": "#ECF2F5"
    },```
  Of course it's possible to add there additional fields variable name, but I think is not worth now do this for removing one condition in the view.

Reply:
sorry, I forgot it's json

Reply:
On the other hand, it's inconsistency in displayed value

![Zaznaczenie_717](https://user-images.githubusercontent.com/2725894/159273402-b6b542f5-63e6-43b6-a146-3ebbbcc119a5.png)

For some colors we use variable name which cannot be used directly in code and for others we have `rgba()` signature. It can be confusing, why sometimes it should be directly copied and sometimes modified.

Reply:
Fixed, now all can be copied to dev tools as value.